src/porepy/numerics/time_step_control.py

```diff
@@ -251,39 +324,49 @@ def next_time_step(
             raise ValueError(msg)
 
         # If iters < max_iter. Proceed to determine the next time step using the
-        # following criteria.
-        # If iters is less than the lower optimal iteration range "iter_low", we can relax
-        # the time step, and multiply by a lower multiplication factor greater than 1,
-        # i.e., "factor_low". If the number of iterations is greater than the upper optimal
-        # iteration range "iter_upp", we have to decrease the time step by multiplying by an
-        # upper multiplication factor smaller than 1, i.e., "factor_upp". If neither of these
-        # situations occur, then the number iterations lies between the optimal iteration
-        # range and the time step remains unchanged.
-        if iterations <= self.iter_low:
+        # following criteria:
+        #     (C1) If iters is less than the lower optimal iteration range "iter_low",
+        #     we can relax the time step, and multiply by a lower multiplication factor
+        #     greater than 1, i.e., "factor_low".
```

Comment:
again, the terminology is confusing - in my mind, low maps to lowering of the time step, rather than elevating it (that is, the time step is too low). Not sure about what is the best solution here.